CMakeLists.txt

```diff
@@ -364,38 +371,40 @@ install(
   RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/backends/onnxruntime
 )
 
-install(
-  DIRECTORY
-    ${CMAKE_CURRENT_BINARY_DIR}/onnxruntime/
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/backends/onnxruntime
-  PATTERN *lib EXCLUDE
-  PATTERN *bin EXCLUDE
-  PATTERN *include EXCLUDE
-  PATTERN *test EXCLUDE
-)
-
-install(
-  DIRECTORY
-    ${CMAKE_CURRENT_BINARY_DIR}/onnxruntime/bin/
-  USE_SOURCE_PERMISSIONS
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/backends/onnxruntime
-)
-
-if (NOT WIN32)
+if(TRITON_ONNXRUNTIME_DOCKER_BUILD)
```

Comment:
You shouldn't disable install for non-docker build.

Reply:
This install copies over the core onnxruntime libraries and not the backend itself. For Jetson we already built these in advance. 
I will add a comment that clarifies the same.